### PR TITLE
Integration with Tone

### PR DESCRIPTION
**Pull request by vibhasinghal**
Submitted code shows integration of car demo with tone.
If the user enters statements like - I am feeling sad, I am frustrated, I am happy, then user emotion is detected and accordingly a genre of music is played.

In implementation:
1) For every user turn, tone is invoked to setup context variables. This is followed by call to conversation
2) If developer want to disable tone integration, then they need not provide TONE_ANALYZER_USERNAME (in local .env) or link TONE_ANALYZER service on bluemix.

### CODE REVIEW

**Comment by germanattanasio** [comment]
@vibhasinghal you need to run `npm run lint` and fix your lint issues before this can be reviewed.

**Comment by germanattanasio** [comment]
@vibhasinghal did you talk to the Conversation OM  about this?.
This is their public demo.

**Comment by vibhasinghal** [comment]
Yes .. am working with Mitch on this. However, do not deploy yet. I'll let you know once we are ready for the push.

File: README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ understands that in both cases your intent is the same and responds accordingly.
 ## Deploy the App
 1 Click this button to Deploy to Bluemix.
 
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; [![Deploy to Bluemix](https://bluemix.net/deploy/button.png)](https://bluemix.net/deploy?repository=https://github.com/watson-developer-cloud/car-dashboard)
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; [![Deploy to Bluemix](https://bluemix.net/deploy/button.png)](https://bluemix.net/deploy?repository=https://github.com/vibhasinghal/car-dashboard)
```
Comment (germanattanasio) [inline README.md]: This change should be removed

File: addons/.gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+node_modules
```
Comment (germanattanasio) [inline addons/.gitignore]: `.gitignore` should not be here

File: app.js [hunk 3]
```diff
@@ -50,13 +50,27 @@ app.use ( bodyParser.json () );
...
+  version_date: '2016-07-11'
 } );
 
+/**
+ * Instantiate the Watson Tone Analyzer Service
+ */
+
+var toneAddon = true;
+
+var tone_detection = require("./addons/tone_detection.js"); //required for tone detection
```
Comment (germanattanasio) [inline app.js]: use `'` not `"`

**Comment by vibhasinghal** [comment]
No longer required.

**Pull request closed without merging**